tensorflow/tools/pip_package/setup.py

```diff
@@ -57,8 +57,8 @@
     'numpy >= 1.14.5, < 2.0',
     'six >= 1.10.0',
     'protobuf >= 3.6.1',
-    'tensorboard >= 1.13.0, < 1.14.0',
-    'tensorflow_estimator >= 1.13.0rc0, < 1.14.0rc0',
+    'tb-nightly >= 1.14.0a20190301, < 1.14.0a20190302',
+    'tensorflow_estimator >= 2.0.0-alpha0, < 2.1.0-rc0',
```

Comment:
This version spec is unsatisfiable - should the dashes be there?  It think PyPI versions do not allow those?

```shell
$ pip install 'tensorflow_estimator >= 2.0.0-alpha0, < 2.1.0-rc0'
Collecting tensorflow_estimator<2.1.0-rc0,>=2.0.0-alpha0
  Could not find a version that satisfies the requirement tensorflow_estimator<2.1.0-rc0,>=2.0.0-alpha0 (from versions: 1.10.6, 1.10.7, 1.10.8, 1.10.9, 1.10.10, 1.10.11, 1.10.12, 1.13.0rc0, 1.13.0)
No matching distribution found for tensorflow_estimator<2.1.0-rc0,>=2.0.0-alpha0
```